### ISSUE TITLE
ReXGroundingCT data

### ISSUE THREAD

**Issue by sanjayss34** [MIC-DKFZ/VoxTell#11]
Thanks for releasing your great work! Is ReXGroundingCT included in the training data of the checkpoint (v1.1) that you have publicly released? I don't see it listed in the datasets in Tables 4 and 5, but the dice score of 0.22 at https://github.com/MIC-DKFZ/VoxTell/issues/5 suggests that perhaps the model is trained on the data? If ReXGroundingCT is part of the training data for v1.1, then can you explain what subset of ReXGroundingCT is included in the training data (e.g. is it all of the 2,992 cases)?

**Comment by mrokuss** [comment]
Hey @sanjayss34,

thanks for the kind words! To clarify: yes, ReXGroundingCT is partially included in the training data of the publicly released v1.1 checkpoint (not in the v1.0 checkpoint), which is why you see a non-trivial Dice of 0.22 out of the box. However, the model was not specifically optimized for the benchmark setting.

The score of 0.28 we report in the paper corresponds to a version of VoxTell that was additionally fine-tuned on the full ReXGroundingCT training split. We also achieved roughly the same performance on the [official leaderboard](https://rexrank.ai/ReXGroundingCT/index.html). Here's a quick overview:

| Model | Dice |
|---|---|
| SAT (fine-tuned) | 13.1 |
| VoxTell v1.1 (general) | 22.3 |
| VoxTell (fine-tuned, paper) | 28.2 |

So v1.1 is a general-purpose model meant for broad concept coverage, not a benchmark-specific checkpoint. We may release the fine-tuned version at a later stage. Hope that clears things up!

Max

**Comment by sanjayss34** [comment]
Thanks, Max! Could you please share the details on how to reproduce the fine-tuning, starting from the v1.1 checkpoint? I know you provided the list of datasets and how you processed them in the appendix, but what about learning rate schedule, batch size, number of steps, and other important details?

**Comment by moritzlangenberg** [comment]
Hey @sanjayss34 ,
we use SGD with an initial learning rate of 1 × 10−4 and polynomial decay, batch size of 2 and train for 1000 epochs and otherwise we train as described in the paper. Are there any specific details you miss or where we can help out? 

Best,
Moritz

**Comment by edomerli** [comment]
Hi @moritzlangenberg,
first of all, congrats on the great work!

I had a question regarding this:
> The score of 0.28 we report in the paper corresponds to a version of VoxTell that was additionally fine-tuned on the full ReXGroundingCT training split.

You use a similar phrasing in #5 as well.

However, I understand from section B.3 of the paper that the fine-tuning dataset also included your curated instance-specific dataset from public sources, namely:

<img width="700" height="300" alt="Image" src="https://github.com/user-attachments/assets/f2726620-8734-4c90-be45-23bc7be0f0ea" />

<br/>

Did you try also fine-tuning on ReXGroundingCT training split only (without the instance-specific dataset part)? If so, did it score close to 0.28?
I'm trying to get a rough estimate of the amount of data required for fine-tuning to work.

Thanks! :)

**Comment by moritzlangenberg** [comment]
Hi @edomerli,
Thanks a lot, happy to hear you like the work!
No, we didn't run that ablation, we never fine-tuned on the ReXGroundingCT training split alone. As you correctly noted, the fine-tuning mix always included our curated instance-specific data alongside ReXGroundingCT. The reason is that the goal of this stage was not to optimize for the ReXGroundingCT benchmark specifically, but to strengthen the model's general instance-level capabilities, and ReXGroundingCT was simply one component of that broader mix.

Best,
Moritz

**Comment by moritzlangenberg** [comment]
@sanjayss34 any update on this?
